nlmod/dims/base.py

```diff
@@ -125,6 +126,9 @@ def to_model_ds(
     transport : bool, optional
         flag indicating whether dataset includes data for a groundwater
         transport model (GWT). Default is False, no transport.
+    remove_nan_layers : bool, optional
```

Comment:
we should probably rename this to 'remove_inactive_layers' or 'remove_zero_thickness_layers'.

With the move to internally always assume modflow-style layer models (with one top and N bottoms), renaming this kwarg would make it more clear what we're doing. 

We do need to maintain some backward compatibility though, so we should add the new name and throw a DeprecationWarning for the old one. Up to you whether you open a new issue for this or add it as part of this PR :).

Reply:
#300